src/domainPaths/domainPathRouter.ts

```diff
@@ -0,0 +1,21 @@
+import { IDomainPath, IDomainPathRouter } from '../types'
+
+export class DomainPathRouter implements IDomainPathRouter {
+  domainMap: {[url: string ]: IDomainPath}
+
+  constructor (domainMap: {[url: string ]: IDomainPath}) {
+    this.domainMap = domainMap
+  }
+
+  // eslint-disable-next-line no-warning-comments
+  // TODO - try to get service whose URL matches and is the longest(most specific)?
+  getDomainPathForUrl (url: string): IDomainPath | undefined {
```

Comment:
The name is a bit unruly, I find it quite hard to read when skimming code. Is there another way to get a domainPath other than via an URL?

Can we shorten it to `getDomainPath`?

Reply:
Done.